int32 SMinToEmpty
int32 SPerCharge

int32 B1RelativeStateOfCharge
float32 B1Voltage
float32 B1Current
float32 B1RemainingCapacity
float32 B1Temp #degrees celsius
int32 B1AvgTimeToEmpty
string B1Status

int32 B2RelativeStateOfCharge
float32 B2Voltage
float32 B2Current
float32 B2RemainingCapacity
float32 B2Temp #degrees celsius
int32 B2AvgTimeToEmpty
string B2Status
